crates/uv/src/commands/auth/mod.rs

```diff
@@ -0,0 +1 @@
+pub(crate) mod configure;
```

Comment:
At first I thought this would need to be split out, but I'll probably update this to use just `commands/auth.rs`